plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml

```diff
@@ -36,24 +37,6 @@ Component
             name: "cura"
         }
 
-        LinearGradient
-        {
-            anchors.fill: parent
-            gradient: Gradient
-            {
-                GradientStop
-                {
-                    position: 0.0
-                    color: "#f6f6f6" // TODO: Theme!
-                }
-                GradientStop
-                {
-                    position: 1.0
-                    color: "#ffffff" // TODO: Theme!
-                }
-            }
-        }
-
```

Comment:
Is this deletion intentional?

Reply:
Yes, the gradient background did not work well with the fades on the carousel so we decided to drop them.